Formula/lepton.rb

```diff
@@ -0,0 +1,23 @@
+class Lepton < Formula
+  desc "Tool and file format for losslessly compressing JPEGs"
+  homepage "https://github.com/dropbox/lepton"
+  url "https://github.com/dropbox/lepton/archive/1.0.tar.gz"
+  sha256 "2ec19eee0f4e78de4cb6a1d48c2cb2615de77abeb1bb22c17d8cf90cd725046a"
+
+  depends_on "cmake" => :build
+
+  def install
+    # Wrong header, fixed in https://github.com/dropbox/lepton/commit/d122450
+    inreplace "src/vp8/util/memory.cc", "<emmintrin.h>", "<immintrin.h>"
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    bin.install "lepton", "lepton-avx"
```

Comment:
That one already landed, but there are more upstream issues to be resolved. At least part of the CI failures appear to be due to the Mac minis used by the test bot not supporting SSE 4.1 instructions — see https://github.com/dropbox/lepton/issues/11 (I have no idea why that's the case with the Ivy Bridge CPUs). [The maintainer promised to improve portability, and there are already patches to review](https://github.com/dropbox/lepton/issues/11#issuecomment-233120456), although the author refused to send patches in the form of a PR. There's no ETA for a fix, but I'm positive that a fix will be made available sooner or later, and with it might even come a new release that wraps up everything. I'll wait until then before applying patches as necessary.